onnx/defs/nn/defs.cc

```diff
@@ -3447,6 +3449,13 @@ ONNX_OPERATOR_SET_SCHEMA(
             "past state cache for value with shape `(batch_size, kv_num_heads, past_sequence_length, v_head_size)`",
             "T2",
             OpSchema::Optional)
+        .Input(
+            6,
+            "kv_nonpad_seqlen",
```

Comment:
Another candidate for the name is "dynamic_kv_seqlen" ... or maybe "nonpadded_kv_seqlen" would be better.

Reply:
Updated to "nonpad_kv_seqlen" for now.